src/react/src/util/util.js

```diff
@@ -1481,3 +1482,39 @@ export const getLastPathParameter = url => {
     const cleanUrl = url.split('?')[0];
     return last(split(trimEnd(cleanUrl, '/'), '/')) || '';
 };
+
+export const isRequiredFieldValid = val => !isEmpty(trim(val));
+
+export const getSelectStyles = (isErrorState = false) => ({
```

Comment:
Suggestion: we have `src/react/src/__tests__/utils.tests.js` where we test utility functions. That would be great to have some tests for this function there as well, but I leave this decision up to you, feel free to skip. Good work, approved.

Reply:
Thanks, Vadim.
I decided to add tests for the isRequiredFieldValid and getSelectStyles functions.

Reply:
Thanks, Olek.